### PR TITLE
add tests for the Map

### PR DESCRIPTION
**Pull request by AdrienRem**

### CODE REVIEW

**Review by TheCrazyKing — APPROVED** [review]
I don't see any inconsistencies. Looking forward to see some coverage on the master branch !

File: app/src/androidTest/java/ch/epfl/sdp/map/MapActivityTest.java [hunk 1]
```diff
@@ -31,10 +40,71 @@
...
+
+        while (sentinel.get() == 0){sleep(300);}
+        sentinel.set(0);
+
+        activityRule.runOnUiThread(() -> {
+            mapFragment.getMap().getStyle(style -> {
+                Layer layer = style.getLayer(HeatMapHandler.HEATMAP_LAYER_ID);
+                assertNotNull(layer);
+                assertEquals(VISIBLE, layer.getVisibility().getValue());
+            });
```
Comment (lucieho) [inline app/src/androidTest/java/ch/epfl/sdp/map/MapActivityTest.java:104]: You might want to consider extracting a method from this lambda used similarly in the next test :)

**Review by lucieho — APPROVED** [review]
Very nice, it's great that you could find a way to test the map!

File: app/src/androidTest/java/ch/epfl/sdp/map/MapActivityTest.java (hunk 1 above)
Comment (AdrienRem) [reply app/src/androidTest/java/ch/epfl/sdp/map/MapActivityTest.java:104]: Done in my last commit !

**Pull request merged**